Hymns/Audio Player/AudioPlayerView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import AVFoundation
+import Combine
+
+enum PlaybackState: Int {
+    case waitingForSelection
+    case buffering
+    case playing
+}
+
+struct AudioView: View {
+
+    let player = AVPlayer()
+    let item: URL?
+    fileprivate let seekDuration: Float64 = 5
+    fileprivate var playerCurrentTime: Float64 {
+        CMTimeGetSeconds(self.player.currentTime())
+    }
+
+    @State var currentlyPlaying = true
+    @State var playbackState: PlaybackState = .waitingForSelection
+
+    var body: some View {
+        VStack {
+            AudioPlayerControlsView(player: player,
+                                    timeObserver: PlayerTimeObserver(player: player),
+                                    itemObserver: PlayerItemObserver(player: player), playbackState: $playbackState)
+            HStack(spacing: 30) {
+                //Reset button currently needed for when you tab to chords, piano, or guitar....
```

Comment:
If the audio player is broken when you change tabs, this seems like a bug.

If you don't want to fix it now, then we need to track it somehow so we don't forget about it. I suggest putting a a TODO and making a Github issue

Reply:
I like the idea of moving to a TODO for now. I added in commit 3678e3a and will add an issue now

Reply:
This issue got fixed when we moved the audio player to the bottom toolbar